src/test/unit/marker-data.test.js

```diff
@@ -377,21 +385,21 @@ describe('filterRawMarkerTableToRange', () => {
       ['6', 8, { type: 'tracing', interval: 'end' }],
     ];
 
-    const thread = setup(markers);
-    const filteredMarkerTable = filterRawMarkerTableToRange(
-      thread.markers,
-      2.3,
-      5.6
-    );
+    const originalThread = setup(markers);
+
+    const filteredThread = {
+      ...originalThread,
+      markers: filterRawMarkerTableToRange(originalThread.markers, 2.3, 5.6),
+    };
+
+    const threadRange = getTimeRangeForThread(filteredThread, 1);
```

Comment:
This change is copied over and over below.
Maybe it or part of it could have gone to `setup`...